code/__DEFINES/flags.dm

```diff
@@ -23,6 +23,7 @@
 
 #define BLOCK_GAS_SMOKE_EFFECT	8192	// blocks the effect that chemical clouds would have on a mob --glasses, mask and helmets ONLY!
 #define THICKMATERIAL 			8192	//prevents syringes, parapens and hypos if the external suit or helmet (if targeting head) has this flag. Example: space suits, biosuit, bombsuits, thick suits that cover your body. (NOTE: flag shared with BLOCK_GAS_SMOKE_EFFECT)
+#define AUTOEXTINGUISH			16384	// Suits that have this will automatically extinguish a burning mob when worn.
```

Comment:
Having an entire flag for one thing in clothing when we're super low on flags is badddd.